logger.go

```diff
@@ -21,8 +33,43 @@ func NewLogger() (*zap.Logger, error) {
 	var err error
 
 	loggerOnce.Do(func() {
-		Logger, err = zap.NewProduction()
+		lumberJackLogger := lumberjack.Logger{
+			Filename:   "logs/server.log", //Filename is the file to write logs to
+			MaxSize:    50,                //MB
+			MaxBackups: 30,                //MaxBackups is the maximum number of old log files to retain.
+			MaxAge:     90,                //days
+			Compress:   false,
+		}
+		zap.RegisterSink("lumberjack", func(*url.URL) (zap.Sink, error) {
```

Comment:
Error check is missing, could you please assign top-level err variable here then return?

Reply:
OK, I will refactor code and check zap.RegisterSink( ) returned err